public/content/roadmap/dencun/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Cancun-Deneb (Dencun) FAQ
+description: Frequently asked questions regarding the Cancun-Deneb (Dencun) network upgrade
+lang: en
+---
+
+# Cancun-Deneb (Dencun) {#dencun}
+
+Cancun-Deneb (Dencun) is an upgrade to the Ethereum network which activates **Proto-Danksharding (EIP-4844)**, introducing data **blobs** for cheaper layer 2 (L2) rollup storage.
```

Comment:
```suggestion
Cancun-Deneb (Dencun) is an upgrade to the Ethereum network which activates **Proto-Danksharding (EIP-4844)**, introducing ephemeral data **blobs** for cheaper layer 2 (L2) rollup storage.
```

to highlight their unique property

Reply:
Only push back here is assuming readers will know what ephemeral is; Was looking to use a more descriptive word here